src/_base/application/overlay/Jenkinsfile.twig

```diff
@@ -6,9 +6,15 @@ pipeline {
     }
     triggers { cron(env.BRANCH_NAME == '{{ @('git.default_branch') }}' ? 'H H(0-6) * * *' : '') }
     stages {
-        stage('Build') {
+        stage('Setup') {
             steps {
-                sh 'ws install'
+                // ws install without enabling
+                sh 'ws install --step download'
+                sh 'ws install --step overlay'
+                sh 'ws install --step validate'
+                sh 'ws install --step prepare'
```

Comment:
could we rather use `ws harness download` and `ws harness prepare` instead of individual steps? or does that not cover all of these steps?